x-pack/plugins/security_solution/cypress/tasks/rule_details.ts

```diff
@@ -158,3 +159,7 @@ export const hasIndexPatterns = (indexPatterns: string) => {
 export const goToRuleEditSettings = () => {
   cy.get(EDIT_RULE_SETTINGS_LINK).click();
 };
+
+export const assertRowsAreHighlighted = () => {
```

Comment:
`non-issue` - Glo discourages to put assertions in the tasks and emphasizes to be directly in the tests, specially since it is one line.

Honestly, I am not sure why is that? May a convention. So I will leave it upto you.

Reply:
Haha, didn't know that. I added the assertion here since there is another assertion in this file. But will switch to the inline check. It's more concise imo. Thanks!